app/api/resources/mentorship_relation.py

```diff
@@ -59,6 +62,20 @@ def is_valid_data(data):
 
         return {}
 
+    @staticmethod
+    def send_email_request(user_id,data):
+        print(data)
```

Comment:
this print statement should not be here for merging, this was just for testing purposes, right?